File: PERL/Cap05/c05ex10.pl
# c05ex10.pl

#!/usr/bin/perl

use strict;
use warnings;

my $RESP = "S";
inicio:
  printf "Entre um valor numerico: ";
  my $N = <STDIN>;
  chomp $N;

  my $R = $N * 3;

  print "Resultado = $R\n";
  print "\n";

  printf "Deseja continuar? (S/N): ";
  $RESP = <STDIN>;
  chomp $RESP;

  print "\n";  
  if (uc($RESP) ne "S") {goto fim};
  
goto inicio;
fim:

print "\n";
print "Tecle <Enter> para encerrar... ";
<STDIN>;

File: RUBY/Cap05/c05ex10.rb
# c05ex10.rb

#!/usr/bin/env ruby

resp = "S"
loop do
  print "Entre um valor numerico: "
  n = gets.chomp.to_i

  r = n * 3

  puts "Resultado = #{r}"
  puts

  print "Deseja continuar? (S/N): "
  resp = gets.chomp

  break if (resp.upcase != "S")
  puts 
end 

puts 
print "\nTecle <Enter> para encerrar... "
gets
